### PR TITLE
DominateSay

### PR DESCRIPTION
**Pull request by surepy**
"todo - Chat and Related - dominatesay"

totally did not copypaste killsay

### CODE REVIEW

**Comment by BenCat07** [comment]
no, you changed submodules that needed no changes

**Comment by BenCat07** [comment]
oh you undid it again

File: src/hacks/DominateSay.cpp [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+//	a much better default dominatesay would be appreciated.
+const std::vector<std::string> builtin_default = {
+    "dominating %name%! (%dominum% dominations)",
+    "%name%, getting tapped?",
+    "%killer% is dominating the server with %dominum% dominations!",
+};
+
+//	same goes to this one
+const std::vector<std::string> jp_anime = {
```
Comment (TotallyNotElite) [inline src/hacks/DominateSay.cpp]: wtf did i just read xd

Comment (surepy) [reply src/hacks/DominateSay.cpp]: I'm not creative ok

File: src/hacks/DominateSay.cpp [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+ *
+ *	Created on: October 30, 2018
+ */
+
+#include <hacks/DominateSay.hpp>
+#include <settings/Int.hpp>
+#include "common.hpp"
+
+static settings::Int dominatesay_mode{ "dominatesay.mode", "0" };
+static settings::String filename{ "dominatesay.file", "dominatesay.txt" };
```
[the remark this answers is not in the document]
Comment (surepy) [reply src/hacks/DominateSay.cpp]: I mean it just gives the user more options 

Will do

**Comment by TotallyNotElite** [comment]
Thanks!

**Pull request merged**